RetailCoder.VBE/UI/FindSymbol/FindSymbolDialog.cs

```diff
@@ -26,6 +26,8 @@ private void viewModel_Navigate(object sender, NavigateCodeEventArgs e)
         public FindSymbolDialog()
         {
             InitializeComponent();
+
+            this.Text = string.Format("Rubberduck - {0}", RubberduckUI.FindSymbolDialog_Caption);
```

Comment:
I'm with @Inopiae on this. "Rubberduck" shouldn't be part of the translation string. I'm good with this, but we'll likely need to fix other languages now...